examples/fast_neural_style/README.md

```diff
@@ -0,0 +1,76 @@
+# fast-neural-style
+
+### Introduction
+This example is ported over from [pytorch-examples](https://github.com/pytorch/examples/tree/master/fast_neural_style).
+
+It uses pytorch-ignite to implement an algorithm for artistic style transfer as described in [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155).
+
+This repo gives examples on how to create a Progress Bar with `pytorch-ignite`. 
```

Comment:
What do you mean by : "how to create a Progress Bar with `pytorch-ignite`" ?

Reply:
My bad, I can remove that. I added some functions in the code to create a progress bar to print metrics related to training. It was an additional example of how to use Events.